cert/cert.go

```diff
@@ -292,3 +292,18 @@ func ParseCertAndKey(certPEM, keyPEM string) (*x509.Certificate, *rsa.PrivateKey
 	}
 	return cert, key, nil
 }
+
+// Generate public key from keyPEM string
+func ComputePublicKey(certPEM string) (string, error){
+	cert, err := ParseCert(certPEM)
+	if err != nil {
+		return "", err
+	}
+	pubKey := cert.PublicKey.(interface {})
+	marshalledPubKey, _ := x509.MarshalPKIXPublicKey(pubKey)
```

Comment:
what is the other ret value of MarshalPKIXPublicKey and why is it being ignored?